### DISCUSSION TITLE
Labeling Options with SKOS

### DISCUSSION THREAD

**Discussion by Sideboard** [FAIRmat-NFDI/fairmat-taxonomy#11] [General]
Every resource in the taxonomy should have at least one label. The simplest way is `rdfs:label`. It can even be used multiple times for the same resource, for example synonyms, different languages (though I assume we only use English), etc. 

If there are multiple, it's difficult to decide which one is the main one. The SKOS vocabulary provides several subproperties for this:

- `skos:prefLabel` "preferred label", restriction: unique for each language tag
- `skos:altLabel` "alternative label", e.g. synonyms
- `skos:hiddenLabel` "hidden label", not supposed to be shown but usable for searches, e.g. common misspellings

The question are:

1. Is one `rdfs:label` enough for our use case?
2. If not, do we want to use the extended ones from SKOS?

#### Examples (from the [SKOS Reference](https://www.w3.org/TR/skos-reference/#labels))

General usage
```
 <MyResource>
  skos:prefLabel "animals"@en ;
  skos:altLabel "fauna"@en ;
  skos:hiddenLabel "aminals"@en ;
  skos:prefLabel "animaux"@fr ;
  skos:altLabel "faune"@fr .
```

prefLabel is not necessary but recommended
```
<Love> skos:altLabel "adoration"@en , "desire"@en .
```

Language sub-division is possible but probably too much for most purposes
```
 <Colour> skos:prefLabel "color"@en , "color"@en-US , "colour"@en-GB .
```

**Comment by JosePizarro3** [comment]
I am just going to comment here what I asked in person:

We could enforce having `skos:prefLabel` as the label defining the property name as well as the IRI. This way, we make sure that we can disregard the IRI, which is a source of bugs when dealing with platforms such as Protegé. 

If this attribute does not exist for a property, then an error should pop up (when exporting this into the web or the NOMAD metainfo defs). This can be discussed, but I felt that the IRI might be a very big source of errors if some fallback is given to it (just take the example of `ElectronicTransportProperty` and `TransportProperty` as defined in the Protegé project).
